docs/content/en/_index.md

```diff
@@ -12,46 +12,51 @@ description: "K8ssandra documentation: architecture, configuration, guided tasks
+K8ssandra Operator is the most recent implementation that allows you to deploy Cassandra into multi-cluster and multi-region Kubernetes environments.
 
-* All Cassandra containers are preinstalled with:
-  * [Metrics Collector for Apache Cassandra](https://github.com/datastax/metric-collector-for-apache-cassandra)
-  * [Management API for Apache Cassandra](https://github.com/datastax/management-api-for-apache-cassandra)
-* Cass Operator to support management tasks in Kubernetes. 
-* Prometheus Operator `ServiceMonitor` custom resources complete with metric labeling.
-* Grafana Operator `Dashboard` custom resources configured with metrics exposed by Prometheus.
-* Reaper to repair Cassandra data.
-* Medusa for backup/restore operations.
+## Why do I want K8ssandra Operator?
 
-## Next steps
+Apache Cassandra is _the_ NoSQL database for applications that require resilience and scalability. Unfortunately this comes with the same burdens as other distributed systems. There are multiple nodes replicating data all the time. Understanding the health of these systems requires advanced tooling and knowledge of the constituent parts. Users could spend time investigating and building out solutions to ensure operational stability of their Cassandra clusters. K8ssandra Operator looks to provide those integrations from the start in a simple easy to deploy package.
 
-* [FAQs]({{< relref "faqs" >}}): If you're new to K8ssandra, these FAQs are for you. 
-* [Install]({{< relref "install" >}}): K8ssandra install steps for local development or production-ready cloud platforms.
-* [Quickstarts]({{< relref "quickstarts" >}}): Post-install K8ssandra topics for developers or Site Reliability Engineers.
-* [Components]({{< relref "components" >}}): Dig in to each deployed component of the K8ssandra stack and see how it communicates with the others.
-* [Tasks]({{< relref "tasks" >}}): Need to get something done? Check out the Tasks topics for a helpful collection of outcome-based solutions.
-* [Reference]({{< relref "reference" >}}): Explore the K8ssandra configuration interface (Helm charts), the available options, and a Glossary.
+## What is K8ssandra Operator good for?
+
+K8ssandra Operator is a great fit for developers and Site Reliability Engineers (SREs) looking for easy to install and manage Cassandra clusters. Even if your environment currently does not run Cassandra on Kubernetes, we believe that simple installation and upkeep will win you over. 
+
+## Next steps
 
-We encourage you to actively participate in the [K8ssandra community](https://k8ssandra.io/community/).
+* If you want to deploy K8ssandra Operator and explore single- and multi-cluster deployment examples, start in its local dev (kind) install topic on the [docs-v2](https://docs-staging-v2.k8ssandra.io/install/local/) area of our site.
```

Comment:
I see what you were saying about replacing links... this is going to be annoying going from one version of the docs to another.  I guess after this initial push it will stabilize and be easier, sorry about that!